src/model/model.js

```diff
@@ -296,15 +296,44 @@ export default class Model {
 	 *		// which has a loosened schema.
 	 *		const modelFragment = editor.data.toModel( viewFragment );
 	 *
-	 *		editor.model.insertContent( modelFragment, editor.model.document.selection );
+	 *		editor.model.insertContent( modelFragment );
+	 *
+	 * By default the method will use document selection but it can be also used with position, range or selection instances.
```

Comment:
this method

the document selection

it can also be used with a position, range or selection instance